### DISCUSSION TITLE
Smart Button S200B

### DISCUSSION THREAD

**Discussion by kaltasGit** [petretiandrea/home-assistant-tapo-p100#732] [General]
I can only see Battery and Report interval?
no button Push?

![image](https://github.com/petretiandrea/home-assistant-tapo-p100/assets/62810871/647e143d-b95c-407a-95d0-6d2938785bfa)

**Comment by mcatkins** [comment]
I opened an issue on this #770 - and now I discover it mentioned all over the place. Sorry for the extra noise!
